### ISSUE TITLE
Add `play` function to `Kino.Audio`

### ISSUE THREAD

**Issue by relardev** [livebook-dev/kino#457]
I was looking for a way to notify myself when some event happens, tried implementing `Kino.JS.Live` that sends browser notification but didn't managed to make it work. Ended up using `Kino.Audio` with added functionality for playing the audio from the code:

```
      kino = Kino.Audio.new(...)
      // somewhere else, when some event happens
      Kino.Audio.play(kino)
```

I can open a PR if there is interest in adding this to Kino

**Comment by jonatanklosko** [comment]
Sounds good, a PR is welcome :) In that case it would make sense to pair it with `Kino.Audio.pause/1`.

**Comment by jonatanklosko** [comment]
Closed in #458.
